core/serv/src/main.rs

```diff
@@ -553,6 +553,8 @@ impl ServiceCommand {
                 let api_host_port = rest_api_host_port(api_url.clone());
                 let rest_address = api_host_port.clone();
 
+                ya_net::hybrid::send_bcast_new_neighbour().await;
```

Comment:
After rethinking: this could cause, that we need to wait a few seconds (it's a timeout for finding Nodes on relay) before we start yagna.
We should spawn it.
This is the only important comment in this review

Reply:
resolved, thank you